Update documentation on latest API (#79)
ami-iit/jaxsim: README.md

## Installation

You can install the project using [`conda`][conda]:

```bash

> conda install jaxsim -c conda-forge 

Reviewer: ```suggestion
conda install jaxsim -c conda-forge
```

Reviewer: @flferretti can you setup you editor to remove automatically blank trailing spaces?

```

> Or using [`pypa/pip`][pip], preferably in a [virtual environment][venv]::

Reviewer: ```suggestion
Alternatively, you can use [`pypa/pip`][pip], preferably in a [virtual environment][venv]:
```
